packages/lab/src/static-list/StaticListItemContent.tsx

```diff
@@ -0,0 +1,39 @@
+import { makePrefixer } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { clsx } from "clsx";
+import {
+  type ComponentPropsWithoutRef,
+  type ForwardedRef,
+  type ReactNode,
+  forwardRef,
+} from "react";
+import staticListItemContent from "./StaticListItemContent.css";
+
+const withBaseName = makePrefixer("saltStaticListItemContent");
+
+export interface StaticListItemContentProps
+  extends ComponentPropsWithoutRef<"div"> {
+  /**
+   * The content of Static List Item
+   */
+  children?: ReactNode;
+}
+
+export const StaticListItemContent = forwardRef(function StaticListItemContent(
+  { children, className, ...restProps }: StaticListItemContentProps,
+  ref: ForwardedRef<HTMLDivElement>,
```

Comment:
```suggestion
export const StaticListItemContent = forwardRef<HTMLDivElement, StaticListItemContentProps>(function StaticListItemContent(
  { children, className, ...restProps },
  ref,
```

will need formatting though